core/src/main/kotlin/org/evomaster/core/output/dto/JavaDtoWriter.kt

```diff
@@ -0,0 +1,105 @@
+package org.evomaster.core.output.dto
+
+import org.evomaster.core.output.Lines
+import org.evomaster.core.output.OutputFormat
+import org.evomaster.core.output.TestSuiteFileName
+import org.evomaster.core.utils.StringUtils
+import java.nio.file.Files
+import java.nio.file.Path
+
+class JavaDtoWriter(private val testSuitePath: Path,
+                    private val outputFormat: OutputFormat,
+                    private val dtoClass: DtoClass) {
+
+    val dtoFilename: TestSuiteFileName
+
+    init {
+        dtoFilename = TestSuiteFileName(appendDtoPackage(dtoClass.name))
+    }
+
+    fun write() {
```

Comment:
any reason to create a class with constructor, instead of having an object (eg `object JavaDtoWriter`) with a parametric `fun write(...)` ?

Reply:
Actually, two reasons:
1. Hadn't thought about it, I just got writing and my first impulse was to add that information in the constructor.
2. This writer will most likely change. I was thinking this would change into some kind of delegate/decorator. You should actually ask the `DtoClass` to write itself, which in turn would call the right type of DtoWriter